client/src/Components/App.js

```diff
@@ -14,12 +15,11 @@ class App extends Component {
  
         </header>
         <HeroSection/>
-        
+        <Questionnaier />
```

Comment:
In the future, you could have your react-router code here, and set up `<Questionnaire />` as one of the app's routes :)
i.e. 
```
<Switch>
  ...
  <Route path='/questionnaire' component={Questionnaire} />
  ...
</Switch>
```